src/tests/ftest/util/dmg_utils_base.py

```diff
@@ -77,27 +76,92 @@ def hostlist(self, hostlist):
     def get_sub_command_class(self):
         # pylint: disable=redefined-variable-type
         """Get the dmg sub command object based upon the sub-command."""
-        if self.sub_command.value == "network":
+        if self.sub_command.value == "check":
+            self.sub_command_class = self.CheckSubCommand()
+        elif self.sub_command.value == "config":
+            self.sub_command_class = self.ConfigSubCommand()
+        elif self.sub_command.value == "cont":
+            self.sub_command_class = self.ContSubCommand()
+        elif self.sub_command.value == "network":
             self.sub_command_class = self.NetworkSubCommand()
         elif self.sub_command.value == "pool":
             self.sub_command_class = self.PoolSubCommand()
+        elif self.sub_command.value == "server":
+            self.sub_command_class = self.ServerSubCommand()
         elif self.sub_command.value == "storage":
             self.sub_command_class = self.StorageSubCommand()
         elif self.sub_command.value == "system":
             self.sub_command_class = self.SystemSubCommand()
-        elif self.sub_command.value == "server":
-            self.sub_command_class = self.ServerSubCommand()
-        elif self.sub_command.value == "cont":
-            self.sub_command_class = self.ContSubCommand()
-        elif self.sub_command.value == "config":
-            self.sub_command_class = self.ConfigSubCommand()
         elif self.sub_command.value == "telemetry":
             self.sub_command_class = self.TelemetrySubCommand()
         elif self.sub_command.value == "version":
             self.sub_command_class = self.VersionSubCommand()
         else:
             self.sub_command_class = None
 
+    class CheckSubCommand(CommandWithSubCommand):
+        """Defines an object for the dmg check sub command."""
+
+        def __init__(self):
+            """Create a dmg check subcommand object."""
+            super().__init__("run/dmg/check/*", "check")
+
+        def get_sub_command_class(self):
```

Comment:
As I mentioned on the call this morning, there are two new subcommands here: `enable` and `disable`.  The `dmg check enable` command will take care of getting all of the ranks (re-)started in checker mode and make the system checker mode persistent so that whenever ranks are restarted they stay in checker mode. Running `dmg check disable` will take the system out of checker mode.

Reply:
Done. Thanks.